CMake installation issues

I just wanted to update our docker setup which hasn't been able to build new images in a month and I found prime_server has some problems on several installation routes with our ubuntu 20.04 base image:
With the CMake install from the README, Valhalla wouldn't be able to find libprime_server:
```
-- Checking for module 'libprime_server>=0.6.3'
--   Requested 'libprime_server >= 0.6.3' but version of libprime_server is
CMake Error at /usr/share/cmake-3.16/Modules/FindPkgConfig.cmake:463 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FindPkgConfig.cmake:643 (_pkg_check_modules_internal)
  CMakeLists.txt:208 (pkg_check_modules)
```

The usual **autotools install works** (i.e. valhalla's CMake recognizes it), but I also get timeouts on the tests as in #99 (EDIT: on another clean try, that wasn't the case anymore.. weird..). Curiously enough, the tests do work with CMake build!

Well, I found out what the issue is while writing this :sweat_smile: PR on the way (`VERSION` was not set in CMakeLists, so the pkgconfig template didn't set the version).

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -110,6 +110,7 @@ set(bindir "${CMAKE_INSTALL_PREFIX}/bin")
 set(libdir "${CMAKE_INSTALL_PREFIX}/lib")
 set(includedir "${CMAKE_INSTALL_PREFIX}/include/prime_server")
 set(pkgconfigdir "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig")
+set(VERSION ${PROJECT_VERSION})
 
 install(TARGETS prime_server
 	LIBRARY DESTINATION ${libdir}

